pkg/apis/performanceprofile/v2/performanceprofile_validation.go

```diff
@@ -323,5 +324,11 @@ func (r *PerformanceProfile) validateWorkloadHints() field.ErrorList {
 		}
 	}
 
+	if r.Spec.WorkloadHints.HighPowerConsumption != nil && *r.Spec.WorkloadHints.HighPowerConsumption {
```

Comment:
don'y we need to check if `r.Spec.WorkloadHints != nil`  first to avoid an exception ?

Reply:
I would rap the whole section if so

Reply:
I don't think so, it is done like this in other part of the code

Reply:
I mean there is a check before:
`if r.Spec.WorkloadHints == nil {
		return allErrs
	}`